.github/workflows/toml_ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   toml-module-pass-external-test-suites:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```

Comment:
why?

Reply:
`jq` [is failing](https://github.com/vlang/v/runs/4367807556?check_suite_focus=true#step:11:152) on 20.04 for some unknown reason